libs/books/data-access/src/lib/+state/reading-list.actions.ts

```diff
@@ -41,3 +41,33 @@ export const removeFromReadingListSucceeded = createAction(
   '[Reading List API] Confirmed remove from list',
   props<{ item: ReadingListItem }>()
 );
+
+export const markBookFinished = createAction(
+  '[Reading List API] Mark book finished on list',
+  props<{ item: ReadingListItem }>()
+);
+
+export const markBookFinishedSucceeded = createAction(
+  '[Reading List API] Confirmed mark book finished on list',
+  props<{ item: ReadingListItem }>()
+);
+
+export const markBookFinishedFailed = createAction(
+  '[Reading List API] Failed to mark book finished on list',
+  props<{ item: ReadingListItem }>()
+);
+
+export const markBookNotFinished = createAction(
```

Comment:
This feels like overkill. When would you really do this? I don't think the spec asks for this to be implemented either.

Reply:
I understand the spec did not ask for this. I thought the ability to un-mark a book might be useful at times, for example, if you marked it as finished by mistake or would like to read it again. Since it won't be used very often and you think it's overkill, removing it.